src/details/ArborX_DetailsDistributedSearchTreeImpl.hpp

```diff
@@ -98,7 +98,7 @@ struct DistributedSearchTreeImpl
   static void communicateResultsBack(
       MPI_Comm comm, Kokkos::View<int *, DeviceType> &indices,
       Kokkos::View<int *, DeviceType> offset,
-      Kokkos::View<int *, DeviceType> &ranks,
```

Comment:
What's the motivation for this change?

Reply:
By `&` we usually indicate that the view is getting modified inside. As this is not the case here, it seems reasonable to remove it.

Reply:
Hmm... I think putting `const` somewhere is much clearer but that discussion doesn't belong here.